source/docs/03-metadata.xml

```diff
@@ -1392,9 +1392,19 @@
                         <specDesc key="colophon"/>
                      </specList>
                      </p>
-                  <p>The <gi scheme="MEI">condition</gi> element may contain text, or child elements that may be used for structuring the description (e.g., for linking) to accurately describe the physical state or condition of a source.</p>
-                     <p>The condition can thus be described in general (<gi scheme="MEI">physDesc</gi>) or in specific contexts (see “contained by” section of the element definition).</p>
-                     <p>For describing the condition of parts of a source (e.g., the binding) the <gi scheme="MEI">condition</gi> element can be used on the specific level. The level of detail you provide will depend on your specific needs for accurately recording the material condition.</p>
+                  <p>Encoding general information about the condition of a source.</p>
+                   <p>
+                      <specList>
+                        <specDesc key="condition"/>
+                      </specList>
+                   </p>
+                     <p>Every physical part of a source is in a certain condition. The element <gi scheme="MEI">condition</gi> may contain plain text but also provides several elements that can be used for structuring the text if necessary (e.g. for linking) in order to describe the state or condition of a source.</p>
+                     <p>The condition can thus be described in general (<gi scheme="MEI">physDesc</gi>) or in specific contexts (see element definition “contained by”).</p>
```

Comment:
I'm not sure what this paragraph is supposed to say, or what it is supposed to go with. If it's supposed to go with the previous one it should not be in its own paragraph.

Reply:
The element `condition` can occur at different positions in the MEI-XML-tree. I tried to explain what the differences are when `condition` is used on an higher or lower hierarchical level. 

```suggestion
                     <p>The highest hierarchical level to describe the condition, in general, is at <gi scheme="MEI">physDesc</gi>. For a detailed description of special interest, the element <gi scheme="MEI">condition</gi> can also be used on lower hierarchical levels (see section “contained by” in the element definition).</p>
```

You're right, this shouldn't be in a separate paragraph! I can't fix it here, but I'll do that separately when this is resolved to avoid conflicts.